packages/react-components/src/components/RichTextEditor/RichTextSendBox.tsx

```diff
@@ -369,7 +374,7 @@ export const RichTextSendBox = (props: RichTextSendBoxProps): JSX.Element => {
   const onRenderAttachmentUploads = useCallback(() => {
     return (
       <Stack className={attachmentUploadCardsStyles}>
-        <FluentV9ThemeProvider v8Theme={theme}>
+        <FluentV9ThemeProvider v8Theme={theme} className={customV9Styles.clearBackground}>
```

Comment:
It looks like /* @conditional-compile-remove(file-sharing-acs) */ is missed here

Reply:
good catch, thanks!

Reply:
actually, this entire const is being cc-ed out, so we don't need to cc here: 
<img width="585" alt="image" src="https://github.com/Azure/communication-ui-library/assets/109105353/adf197ce-0903-4669-b45c-ea67bf6978b2">